src/components/claims/claims-list/components/LargeClaimsList.tsx

```diff
@@ -0,0 +1,173 @@
+import styled from '@emotion/styled'
+import { ClaimState } from 'api/generated/graphql'
+import { parseISO } from 'date-fns'
+import formatDate from 'date-fns/format'
+import { useListClaims } from 'graphql/use-list-claims'
+import { LoadingMessage } from 'hedvig-ui/animations/standalone-message'
+import {
+  Table,
+  TableColumn,
+  TableHeader,
+  TablePageSelect,
+  TableRow,
+} from 'hedvig-ui/table'
+import { Monetary, Placeholder } from 'hedvig-ui/typography'
+import React, { useEffect } from 'react'
+import { useHistory } from 'react-router'
+import { useVerticalKeyboardNavigation } from 'utils/keyboard-actions'
+import { getMemberIdColor } from 'utils/member'
+import { useNumberMemberGroups } from 'utils/number-member-groups-context'
+import { convertEnumToTitle, splitOnUpperCase } from 'utils/text'
+
+const ClaimStateBadge = styled.span<{ state: ClaimState }>`
+  display: inline-block;
+  min-width: 8em;
+  text-align: center;
+  padding: 0.3em 1em;
+  background-color: ${({ theme, state }) =>
+    state === ClaimState.Closed ? theme.success : theme.accent};
+  color: ${({ theme }) => theme.accentContrast};
+  font-size: 0.8em;
+  border-radius: 8px;
+`
+
+const TableColumnSubtext = styled.span`
+  font-size: 0.8em;
+  color: ${({ theme }) => theme.semiStrongForeground};
+`
+
+const FlexVertically = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+
+const MemberIdCell = styled(TableColumn)<{
+  memberId: string
+  numberMemberGroups: number
+}>`
+  border-left: 7px solid
+    ${({ memberId, numberMemberGroups }) =>
+      getMemberIdColor(memberId, numberMemberGroups)};
+
+  padding-left: 1em;
+  height: 100%;
+`
+
+export const LargeClaimsList: React.FC<{ page: number }> = ({ page }) => {
+  const history = useHistory()
+  const { numberMemberGroups } = useNumberMemberGroups()
+
+  const [
+    { claims, page: currentPage, totalPages },
+    listClaims,
+    { loading },
+  ] = useListClaims()
+
+  useEffect(() => {
+    listClaims({ page: page - 1 ?? 0 })
+  }, [page])
+
+  const [currentKeyboardNavigationStep] = useVerticalKeyboardNavigation({
+    maxStep: claims.length - 1,
+    onPerformNavigation: (index) => {
+      const claimId = claims[index].id
+      const memberId = claims[index].member?.memberId
+
+      if (!claimId || !memberId) {
+        return
+      }
+
+      history.push(`/claims/${claimId}/members/${memberId}`)
+    },
+  })
+
+  if (loading) {
+    return <LoadingMessage paddingTop={'25vh'} />
+  }
+
+  return (
+    <>
+      <Table>
+        <TableRow>
+          <TableHeader>Member</TableHeader>
+          <TableHeader>Date Registered</TableHeader>
+          <TableHeader>Claim Type</TableHeader>
+          <TableHeader>Claim State</TableHeader>
+          <TableHeader>Claim Reserves</TableHeader>
+        </TableRow>
```

Comment:
If we manage to get `verticalKeyboard` into the Rows "automatically"
```suggestion
        <TableHeader>
          <TableHeaderColumn>Member</TableHeaderColumn>
          <TableHeaderColumn>Date Registered</TableHeaderColumn>
          <TableHeaderColumn>Claim Type</TableHeaderColumn>
          <TableHeaderColumn>Claim State</TableHeaderColumn>
          <TableHeaderColumn>Claim Reserves</TableHeaderColumn>
        </TableRow>
```